tests/http_api/test_http_api.py

```diff
@@ -481,7 +481,7 @@ def test_get_cache_key(connector, auth, data_source):
     data_source.parameters = {'first_name': 'raphael'}
     key = connector.get_cache_key(data_source)
 
-    assert key == 'b35e0189-a527-3f9c-827d-b0c54e23b780'
+    assert key == 'fa95c942-9b94-3f07-9ed4-24c34abfbdae'
```

Comment:
IMO... this is a flaky test, we should probably change this test... that's not the first time i had to update this value... not a blocking point tho... WDYT ?

Reply:
I think the test is valid, it's probably the `get_cache_key` method that should be changed to not depend on a datasource model